src/sql/workbench/browser/designer/designer.ts

```diff
@@ -219,6 +227,17 @@ export class Designer extends Disposable implements IThemable {
 		}
 	}
 
+	private removeTableSelectionStyles(): void {
+		this._styles.tableStyles.listActiveSelectionBackground = undefined;
```

Comment:
this is needed to not show the row selection visual effect?

Reply:
no, because it gives the mismatch blue colors, which is a regression